client/src/components/Chat/ChatScreen.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment } from 'react'
+import React, { useContext } from 'react'
```

Comment:
Again, we are gonna create a new context to import from a file in Contexts so you can remove all of the changes here for now!